### ISSUE TITLE
Openvidu 2.26: release jar missing?

### ISSUE THREAD

**Issue by juhaszp-uhu** [OpenVidu/openvidu#798]
**Describe the bug**
The .jar file for the openvidu-server 2.26 release is missing from github. For previous releases it could be found at e.g. https://github.com/OpenVidu/openvidu/releases/tag/v2.25.0

Is it just an omission, or a new policy that from now on, the release .jar will not be published on github?

**Expected behavior**
That a file exists at https://github.com/OpenVidu/openvidu/releases/tag/v2.26.0/openvidu-server-2.26.0.jar and it contains the openvidu-server release.

**Wrong current behavior**
It doesn't exist.

**Comment by pabloFuente** [comment]
Yes, it is simply a new policy. There is no official OpenVidu deployment that requires direct use of the raw JAR file. Nonetheless, documentation on how to build it from the source code is available here: https://docs.openvidu.io/en/latest/developing/#compiling-openvidu-server
